CHANGELOG.md

```diff
@@ -239,7 +239,7 @@ user manually maintain all the associated guarantees, it now takes a single
 
 * Clarified documentation on a number of `Response` constructors.
 
-* Updated `README` links to point to latest docs on `www.dartdocs.org`.
+* Updated `README` links to point to latest docs.
```

Comment:
These old lines _could_ have been left as is since at the time the update _was_ to this place.

🤷‍♂

Reply:
`repo_manager` has a bit that searches for old strings in packages. This just helps remove false positives